src/compute/gcp/api.clj

```diff
@@ -31,45 +30,51 @@
 
 ;; TODO
 (defn doc-str
-  [{:keys [documentation request required response refs] :as doc}]
-  (when doc
-    (str/join "\n"
-              (cond-> ["-------------------------"
-                       (:name doc)
-                       ""
-                       documentation]
-                request
-                (into [""
-                       "-------------------------"
-                       "Request"
-                       ""
-                       (with-out-str (clojure.pprint/pprint request))])
-                required
-                (into ["Required"
-                       ""
-                       (with-out-str (clojure.pprint/pprint required))])
-                response
-                (into ["-------------------------"
-                       "Response"
-                       ""
-                       (with-out-str (clojure.pprint/pprint response))])
-                refs
-                (into ["-------------------------"
-                       "Given"
-                       ""
-                       (with-out-str (clojure.pprint/pprint refs))])))))
+  [{::descriptor/keys [operation description parameters response] :as doc}]
+  (let [required (keep (fn [[parameter {:strs [required]}]]
+                         (when required
+                           (keyword parameter)))
+                   parameters)]
+    (when doc
+      (str/join "\n"
+        (cond-> ["-------------------------"
+                 operation
+                 ""
+                 description]
+          parameters
+          (into [""
+                 "-------------------------"
+                 "Request"
+                 ""
+                 (with-out-str (clojure.pprint/pprint parameters))])
+          required
+          (into ["Required"
+                 ""
+                 (with-out-str (clojure.pprint/pprint required))])
+          response
+          (into ["-------------------------"
+                 "Response"
+                 ""
+                 (with-out-str (clojure.pprint/pprint response))]))))))
 
 
 (defn doc-data
   [client op]
   (let [op-descriptor (descriptor/get-op-info (::api-descriptor client) op)]
     (-> op-descriptor
-        (select-keys [:description :parameters :responses]))))
-
+      (select-keys [:description :parameters :responses]))))
 
 (defn ops
   [client]
-  (keys (get-in client [::api-descriptor ::descriptor/op->info])))
+  (::descriptor/op->info (::api-descriptor (client-impl/-get-info client))))
```

Comment:
Why this change?

Reply:
to match aws-api behavior

ops returns a map-of op-name op-description, not a coll-of op-name.
https://github.com/cognitect-labs/aws-api/blob/master/src/cognitect/aws/client/api.clj#L163

Reply:
Then it will be used in `doc` function. By doing this, our `doc` implementation looks almost identical aws-api one.
https://github.com/cognitect-labs/aws-api/blob/master/src/cognitect/aws/client/api.clj#L208